src/codec_aom.c

```diff
@@ -97,13 +97,30 @@ static avifBool aomCodecGetNextImage(struct avifCodec * codec,
                                      avifImage * image)
 {
     if (!codec->internal->decoderInitialized) {
+        if (!sample) {
```

Comment:
Just curious (I didn't analyze the code): why do we call this function with a null `sample`? (I see we also check for a null `sample` at line 154 below.)

Reply:
Done.